preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -819,8 +834,17 @@ def process_single(  # noqa: C901
                 message=("No segment aligned."),
             )
         )
+    annotations: dict[str, Any] = {}
```

Comment:
I guess if we are not returning embl flat files (i.e. if embl_file is False in the config) we could also just set the annotations to None to not have to carry them around